template.php

```diff
@@ -681,6 +682,32 @@ function europa_preprocess_block(&$variables) {
       $variables['classes_array'][] = 'link-block';
       $variables['title_attributes_array']['class'][] = 'link-block__title';
       break;
+
+    case 'menu-nexteuropa-site-links':
+      if ($block->region == 'header_top') {
+        $variables['classes_array'][] = 'nexteuropa_site_switcher';
+
+        if ($links = menu_navigation_links('menu-nexteuropa-site-links')) {
+          $i = 0;
+          foreach ($links as $key => $link) {
+            $theone = variable_get('ec_europa_site_switcher' == 'Informational') ? 0 : 1;
```

Comment:
This is broken in my opinion, always comes 1 independently form the variable. Probably a copy paste issue.

Also I would move this out of the loop since it does not change or could be completely avoided if you see my comments in the tools PR: https://github.com/ec-europa/ec-europa-theme-tools/pull/40#discussion_r111173340